phil/s013-course-periods.md

```diff
@@ -54,45 +90,65 @@ need the endpoint to have the same due time / tasking information as HW builder
 
 # Calendar Popup
 
-Decision: Needs discussion w/ UX
-
 ```coffee
-{periods: [ {id: , name: , stats: } ] }
+stats: [ {period_id: "1", ... } ]    # where ... is stats-specific data
 ```
 
-period break up of https://tutor-demo.openstax.org/api/docs/v1/task_plans/stats
+# Class Aggregate review stats
+
+(same as Calendar Popup but with different `...`)
+
+
+# Learning Guide
+
+`GET /courses/1/guides` (or `teacher_guide`?)
+
 ```coffee
-[ { period: { id, name }, stats: ... } ]
+[
+  { period_id: "1", ... }
+  # More periods here
+]
 ```
 
 # Performance Report
 
-rolled up to course?
+rolled up to course? No. Only by period.
 https://tutor-demo.openstax.org/api/docs/v1/courses/performance
 in the export for BE
 
 ```coffee
 [
  {
-   period: { id, name: },
-   data_heading: {},   # same as existing
+   period_id: "1"
+   data_heading: {}   # same as existing
    student_data: []    # same as existing
-  },  # repeat this for each period
+  }
+  # repeat this for each period
 ]
 ```
 
+
 # Roster
 
+`GET /courses/1/roster`
```

Comment:
Currently the endpoint is `/courses/1/students` instead of `/courses/1/roster`.  Would trickle down then to the post, patch, delete.